examples/spotify-api-example/src/main/java/com/spotify/apollo/example/SpotifyApiExample.java

```diff
@@ -0,0 +1,46 @@
+package com.spotify.apollo.example;
+
+import com.spotify.apollo.Environment;
+import com.spotify.apollo.RequestContext;
+import com.spotify.apollo.Response;
+import com.spotify.apollo.httpservice.HttpService;
+import com.spotify.apollo.httpservice.LoadingException;
+import com.spotify.apollo.route.AsyncHandler;
+import com.spotify.apollo.route.Route;
+
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import okio.ByteString;
+
+/**
+ * This example demonstrates how to setup multiple routes in Apollo.
+ *
+ * It uses asynchronous routes that calls the Spotify API to fetch some data and then process it.
+ */
+final class SpotifyApiExample {
+
+  public static void main(String[] args) throws LoadingException {
+    HttpService.boot(SpotifyApiExample::init, "spotify-api-example-service", args);
+  }
+
+  static void init(Environment environment) {
+    AlbumResource albumResource = new AlbumResource();
+    ArtistResource artistResource = new ArtistResource();
+
+    Stream<Route<AsyncHandler<Response<ByteString>>>> routes =
```

Comment:
For some reason I thought versioned routes was not in open source Apollo. Maybe because it isn't mentioned in the docs :)

Will consider this.